libs/damap/src/lib/components/dmp/people/people.component.html

```diff
@@ -50,17 +54,15 @@
 
 <!-- SEARCH -->
 <div [formGroup]="dmpForm" class="mat-search-form-field">
-  <app-person-search [result]="
-      searchResult$ | async
-        | contributorFilter: dmpForm.controls.contributors.value
-    " (termToSearch)="searchContributor($event)" (personToAdd)="addContributor($event)">
+  <app-person-search [searchResult$]="searchResult$" (termToSearch)="searchContributor($event)"
+    (personToAdd)="addContributor($event)">
   </app-person-search>
```

Comment:
question: What is the reason for removing the async pipe and passing an observable into the component?